en/components/calendar.md

```diff
@@ -1,69 +1,81 @@
+Use the Calendar Component to visually represent a date and provide consistent means for navigating its parts through views for days, months and years. The Calendar is meant to be inlined in a view to allow browsing dates and selecting one or more of them e.g. a range of dates. The Calendar is visually identical to the [Ignite UI for Angular Calendar Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/calendar.html) and is used within the [Ignite UI for Angular Date Picker Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/date_picker.html)
 
 ## Calendar Demo
 
 <img class="responsive-img" src="../images/calendar_demo.png" srcset="../images/calendar_demo@2x.png 2x" />
 
 ## Layout
 
-The Calendar supports horizontal and vertical layouts, as well as a base calendar rendering for a simplified display and browsing.
+The Calendar comes in Horizontal or Vertical layout suited to display one month at a time, but also supports a Multiview option for showing two months side by side.
 
 <img class="responsive-img" src="../images/calendar_horizontal.png" srcset="../images/calendar_horizontal@2x.png 2x" />
 <img class="responsive-img" src="../images/calendar_vertical.png" srcset="../images/calendar_vertical@2x.png 2x" />
-<img class="responsive-img" src="../images/calendar_base.png" srcset="../images/calendar_base@2x.png 2x" />
+<img class="responsive-img" src="../images/calendar_multi.png" srcset="../images/calendar_multi@2x.png 2x" />
 
 ## Content
 
-The Calendar supports picking for all three major date portions: the year, the month, and the day. Three content modes are provided, each responsible for the picking of its respective date portion.
+The Calendar can be inserted in day, month or year view to facilitate display of each of the major date portions.
 
-<img class="responsive-img" src="../images/calendar_days.png" srcset="../images/calendar_days@2x.png 2x" />
+<img class="responsive-img" src="../images/calendar_vertical.png" srcset="../images/calendar_vertical@2x.png 2x" />
 <img class="responsive-img" src="../images/calendar_months.png" srcset="../images/calendar_months@2x.png 2x" />
 <img class="responsive-img" src="../images/calendar_years.png" srcset="../images/calendar_years@2x.png 2x" />
 
 ## Week Start
 
-The start of the week is configurable by selecting between the two most common scenarios for the first day: Sunday or Monday.
+The start of the week can also be chosen as an option at the time of inserting a Calendar between the two most common scenarios for first day of the week: Sunday and Monday.
 
 <img class="responsive-img" src="../images/calendar_sun.png" srcset="../images/calendar_sun@2x.png 2x" />
-<img class="responsive-img" src="../images/calendar_mon.png" srcset="../images/calendar_mon@2x.png 2x" />
+<img class="responsive-img" src="../images/calendar_vertical.png" srcset="../images/calendar_vertical@2x.png 2x" />
```

Comment:
```suggestion
<img class="responsive-img" src="../images/calendar_mon.png" srcset="../images/calendar_mon@2x.png 2x" />
```
Maybe we need to change the name of the new image to its previous name to match the other image calendar_sun.png?